### PR TITLE
add api of rpc

### PR DESCRIPTION
**Pull request by Ningsir**
PADDLEPADDLE_PR=45998

https://github.com/PaddlePaddle/Paddle/pull/45998

### CODE REVIEW

**Review by momozi1996 — APPROVED** [review]
测试用的Approve

File: docs/api/paddle/distributed/rpc/rpc_async_cn.rst [hunk 1]
```diff
@@ -0,0 +1,25 @@
+.. _cn_api_distributed_rpc_rpc_async:
+
+rpc_async
+-------------------------------
+
+
+.. py:function:: paddle.distributed.rpc.rpc_async(to, fn, args=None, kwargs=None, timeout=_DEFAULT_RPC_TIMEOUT)
```
Comment (Ligoml) [inline docs/api/paddle/distributed/rpc/rpc_async_cn.rst]:
```suggestion
.. py:function:: paddle.distributed.rpc.rpc_async(to, fn, args=None, kwargs=None, timeout=-1)
```

File: docs/api/paddle/distributed/rpc/rpc_sync_cn.rst [hunk 2]
```diff
@@ -0,0 +1,24 @@
+.. _cn_api_distributed_rpc_rpc_sync:
+
+rpc_sync
+-------------------------------
+
+.. py:function:: paddle.distributed.rpc.rpc_sync(to, fn, args=None, kwargs=None, timeout=_DEFAULT_RPC_TIMEOUT)
```
Comment (Ligoml) [inline docs/api/paddle/distributed/rpc/rpc_sync_cn.rst]:
```suggestion
.. py:function:: paddle.distributed.rpc.rpc_sync(to, fn, args=None, kwargs=None, timeout=-1)
```

**Review by Ligoml — APPROVED** [review]
LGTM for docs

**Pull request merged**